app/views/components/PaginationUtil.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.components
+
+import scala.collection.immutable.Seq
+
+object PaginationUtil {
+
+  def surroundingPages(pageIndex: Int, pageCount: Int, available: Int): Seq[Int] = {
```

Comment:
Can we make this function without vars? just recursive function

Reply:
This class was lifted from binding-tariffs.  I haven't really studied it to understand how it works, but it's tested/proven code and I'd rather just leave it as-is.  Unless we have nothing else to do.